ReplayKitExample/ReplayKitExample/CoreImagePixelBufferInput.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  CoreImagePixelBufferInput.swift
+//  ReplayKitExample
+//
+//  Copyright © 2020 Twilio. All rights reserved.
+//
+
+import CoreImage
+import CoreGraphics
+import CoreVideo
+
+class CoreImagePixelBufferInput {
+    let context = CIContext(options: [CIContextOption.outputColorSpace: NSNull(),
+                                      CIContextOption.workingColorSpace: NSNull()]);
+
+    /// Avoids reallocating memory.
+    var pixelBufferPool: CVPixelBufferPool?
+
+    func createPixelBufferPool(width: Int32, height: Int32, maxBufferCount: Int32) -> CVPixelBufferPool? {
+        print("Creating a CVPixelBufferPool with size=\(width)x\(height), maxBuffers=\(maxBufferCount).")
+        var outputPool: CVPixelBufferPool? = nil
+
+        let poolAttributes: NSDictionary = [
+            kCVPixelBufferPoolMinimumBufferCountKey: 1,
+            kCVPixelBufferPoolAllocationThresholdKey: maxBufferCount
+        ]
+        
+        let pixelBufferAttributes: NSDictionary = [
+            kCVPixelBufferPixelFormatTypeKey: kCVPixelFormatType_420YpCbCr8BiPlanarFullRange,
+            kCVPixelBufferWidthKey: width,
+            kCVPixelBufferHeightKey: height,
+            kCVPixelBufferIOSurfacePropertiesKey: NSDictionary()
+        ]
+        let status = CVPixelBufferPoolCreate(nil,
+                                             poolAttributes,
+                                             pixelBufferAttributes,
+                                             &outputPool)
+        if status != kCVReturnSuccess {
+            // TODO: Throw?
```

Comment:
Plan to complete todo here?

Reply:
Yes, let me take another look at handling the error.